app/src/main/java/com/example/messengerAndroid/ui/contactsViewModel/ContactsViewModel.kt

```diff
@@ -31,17 +31,8 @@ class ContactsViewModel : ViewModel() {
     }
 
     fun addNewUser(name: String, job: String) {
-        Log.d("myTag", "$name")
-        Log.d("myTag", "$job")
         _contactsLiveData.value = _contactsLiveData.value?.toMutableList()?.apply {
-            add(
-                User(
-                _contactsLiveData.value!!.size.toLong(),
-                "...",
-                name,
-                job
-            )
-            )
+            add(User(_contactsLiveData.value!!.size.toLong(), "...", name, job))
```

Comment:
Null-assert need to change with technology that no throw exception.

Reply:
Agree with Stas, that you should replace the null-assert with the Elvis operator or some kind of null check. Also, the hardcoded string should be moved to constants